### PR TITLE
admin checklist email

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #144 #49

### CODE REVIEW

File: server/helpers/mailHelper/checklistReminders/emailBodyToAdmin.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <div style="font-weight: 700;">
+        Intern: <a href="${internURL}">${capitalizeName(internName)}</a>
+        <ul>
+          ${internChecklist.join("\n")}
+        </ul>
+      </div>
+    </p>
+
+    <p>
+      <a href="${bookingURL}">view your booking</a>
```
Comment (RamyAlshurafa) [inline server/helpers/mailHelper/checklistReminders/emailBodyToAdmin.js]: view your booking :x: 
view your booking :heavy_check_mark:

File: server/services/mailing/scheduleReminderEmails.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    });
+  }
+
+  if (reminders.length) {
+    await createScheduledEmails(reminders);
+  } else if (currnetDate.isSameOrBefore(startDate)) {
+    // TODO send email immediately
+  // await sendEmailNow()
+  } else {
+    throw (new Error("start date is in the past"));
```
Comment (MohammedYehia) [inline server/services/mailing/scheduleReminderEmails.js]: no need for the parenthesis 
```suggestion
    throw new Error("start date is in the past");
```

File: server/helpers/mailHelper/checklistReminders/createChecklistArray.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const generateListRow = require("./generateListRow");
+
+
+const createChecklistArray = ({
+  checklist, hostEmail, internEmail,
+}) => {
+  const checklistHtmlRows = checklist.forEach((checklistQuestion) => {
```
Comment (MohammedYehia) [inline server/helpers/mailHelper/checklistReminders/createChecklistArray.js]: this should be `map` not forEach 
```suggestion
  return checklist.map((checklistQuestion) => {
```

File: server/database/data/development/scheduledEmails.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ module.exports = async () => {
 
   const scheduledEmails = [
     {
-      type: "INTERN_REMINDER_7_DAYS",
+      type: "BOOKING_REMINDER_1_WEEK",
```
Comment (MohammedYehia) [inline server/database/data/development/scheduledEmails.js]: I meant this could be on another file as a constant file but it's fine I guess

**Pull request merged**